polling_stations/apps/api/sandbox-responses/AA12AA.json

```diff
@@ -11,7 +11,6 @@
         },
         "type": "Feature"
     },
-    "custom_finder": null,
```

Comment:
Probably not one to deal with in this PR, but I wonder if it is actually useful to keep maintaining sandbox responses on this repo now it is private?